content/docs/projects/trust-manager/api-reference.md

```diff
@@ -428,6 +492,56 @@ Target is the current Target that the Bundle is attempting or has completed sync
 </table>
 
 
+### `Bundle.status.target.additionalFormats`
+
+
+AdditionalFormats specifies any additional formats to write to the target
```

Comment:
self review: having this documented in such detail for the status (as well as the spec above) is probably a little redundant too but it's not the end of the world.